.github/workflows/tests-live.yml

```diff
@@ -43,3 +43,7 @@ jobs:
           LIVE_S3_BUCKET: ${{ secrets.LIVE_S3_BUCKET }}
           AWS_ACCESS_KEY_ID: ${{ secrets.AWS_ACCESS_KEY_ID }}
           AWS_SECRET_ACCESS_KEY: ${{ secrets.AWS_SECRET_ACCESS_KEY }}
+          CUSTOM_S3_BUCKET: ${{CUSTOM_S3_BUCKET}}
+          CUSTOM_S3_KEY_ID: ${{CUSTOM_S3_KEY_ID}}
+          CUSTOM_S3_SECRET_KEY: ${{CUSTOM_S3_SECRET_KEY}}
+          CUSTOM_S3_ENDPOINT: ${{CUSTOM_S3_ENDPOINT}}
```

Comment:
Do these need the same `secrets.`?

Reply:
Those are there already. I thought you added those? 😆

Reply:
I meant should `${{CUSTOM_S3_BUCKET}}` be `${{ secrets.CUSTOM_S3_BUCKET }}`

Reply:
but yes, i did add the secrets!

Reply:
Oh, you are totally right. That was confusing, it declared my workflow invalid, but didn't show it as a failure.